.drone.yml

```diff
@@ -19,8 +19,8 @@ pipeline:
     image: gcr.io/mirrormedia-1470651750304/drone-cloud-sdk:latest
     secrets: [google_credentials]
     commands:
-    - gcloud source repos clone default ../default
-    - cp ../default/tr-project-rest/settings.py ./settings.py
+    - gcloud source repos clone configs ../configs
+    - cp ../configs/mirror-media/tr-projects-rest/prod/settings.py ./settings.py
```

Comment:
Actually this is my fault from the past. `gcloud source repos clone configs ../configs` will clone the configs outside of workspace. This could cause drone server keep the configs in somewhere therefore there might be tricky problems, like no effect on config changes when building.

Ideally we should just copy configs inside drone workspace, like `gcloud source repos clone config`, and take care of the rest, like changing the `COPY` statement in Dockerfile in case we copy redundant files into the dockers.

Reply:
I just remove the repo after config copied, instead of modifying Dockerfile.

Reply:
That works, too. LGTM now. Let's merge this